metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.kafka.controller.metrics;
 
+import org.apache.kafka.common.metrics.MetricConfig;
 import org.apache.kafka.common.utils.Time;
+import org.apache.kafka.raft.internals.TimeRatio;
```

Comment:
Other modules should not import from KRaft's internal types. The best option is to move the TimeRatio type to server-common.